glide.yaml

```diff
@@ -0,0 +1,28 @@
+package: github.com/coreos/locksmith
+import:
+- package: github.com/coreos/etcd
+  version: v2.2.5
+  subpackages:
+  - client
+  - pkg/pathutil
+  - pkg/types
+- package: github.com/coreos/go-systemd
+  version: e3e4f602334ec3e1c2cd68322a1284903c62f940
+  subpackages:
+  - dbus
+  - journal
+  - login1
+- package: github.com/coreos/pkg
+  version: 549bd7890e3564e9a2409d9700ea365005edad8d
+  subpackages:
+  - capnslog
+- package: github.com/godbus/dbus
+  version: 25a4b8ca48c64b3d631110fb74a210f56c401e0b
```

Comment:
Yeah I guess that would be out of scope of this PR. Disregard for now, but it would be nice to bump things to released versions at some point.